### PR TITLE
[FSSDK-9562] prepare for release 5.0.0-beta3

### PR DESCRIPTION
**Pull request by raju-opti**
## Summary
- prepare for release 5.0.0-beta3

## Test plan
- all existing tests should pass
-
## Issues
- FSSDK-9562

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
```
Comment (mikechu-optimizely) [inline CHANGELOG.md:9]: Do we want to mention the re-organization of the repo in the event beta testers find something we missed?

Comment (mikechu-optimizely) [reply CHANGELOG.md:9]: PRs #845 to #851

Comment (mikechu-optimizely) [reply CHANGELOG.md:9]: Maybe also #844 and #841  ?

**Review by mikechu-optimizely — APPROVED** [review]
LGTM though I added some comments/questions.

File: CHANGELOG.md (hunk 1 above)
Comment (raju-opti) [reply CHANGELOG.md:9]: "Do we want to mention the re-organization of the repo in the event beta testers find something we missed?" - I think no. This changelog is for the optimizely-sdk package which just moved to root now. This is not concerned with other packages in the now deprecated monorepo.

Apart from PR 841, others are internal reorganization, I would prefer to skip them. I'll add 841 as it has changes in the behavior of the public api.

**Pull request merged**